ui/ducks/send/send.js

```diff
@@ -1710,6 +1712,9 @@ const slice = createSlice({
             }
           }
         }
+      })
+      .addCase(AMOUNT_MODE_CHANGED, (state, action) => {
```

Comment:
I'm not sure what this new case is doing since doesn't the `amountMode` already get added to the `send` state on line 2420?

Reply:
Not needed, probably just leftover from previous soluitons.